src/CommonDBRelation.php

```diff
@@ -2145,4 +2146,30 @@ public static function getMemberPosition(string $class): int
             return 0;
         }
     }
+
+    public function check($ID, int $right, ?array &$input = null): void
+    {
+        $error_types = [];
+        if (!isset($input[static::$items_id_1]) || empty($input[static::$items_id_1])) {
+            $error_types[] = static::$itemtype_1::getTypeName(1);
+        }
+        if (!isset($input[static::$items_id_2]) || empty($input[static::$items_id_2])) {
+            $error_types[] = static::$itemtype_2::getTypeName(1);
+        }
```

Comment:
If `$mustBeAttached_X` is false, having no value should not be considered as an error.